presto-docs/src/main/sphinx/cache/library.rst

```diff
@@ -0,0 +1,66 @@
+=====================
+Alluxio Cache Library
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+To optimize Presto query latency when reading remote data using Hive connector,
+one can leverage a built-in cache library in each Presto worker.
+This cache, based on Alluixo client-side cache library,
+utilizes the local storage (such as SSD) with configurable capacity and locations,
+and requires no extra service.
+To understand the internals of this local cache and the benchmark results of latency improvement,
+please refer to this `article <https://prestodb.io/blog/2020/06/16/alluxio-datacaching>`_.
+Note that this is a **read-cache**, **completely transparent** from users
+and **fully managed** by each individual Presto worker node.
+To provide Presto an independent, distributed cache service for reads/writes and
+full-featured cache policies, please refer to :doc:`/cache/alluxio`.
+
+
+Setup
+-----
+
+To turn on this Alluxio cache library is very easy:
+simply include the following configuration in ``etc/catalog/hive.properties``
+and restart Presto workers.
```

Comment:
Please note that node selection strategy is a coordinator level config and would require a restart of coordinator to be in effect. I think in the blog I had categorized it between coordinator and worker.

Reply:
updated